Setup mobile version skeleton

KDlamini opened the pull request:
This pull request consists of a feature that setups a mobile version skeleton of a portfolio website with the following:
-  A nav element that is a toolbox containing a logo and menu icon
-  A header element that contains and h1, p, and a elements that depict the heading, description, and social icons respectively.
-  A CSS file with the respective styles. It uses Flexbox and media queries for responsive design.
-  Add header and top navigation background.
- personalize the MIT.md file, add your name, project name, and delete unnecessary lines.
-  Add linters.yml Config file for GitHub Actions.

KDlamini committed:
Create mobile skeleton section:
- add header tag for the top navigation
- add the link to the logo and menu icon
- add logo.png and vector.png files in the images folder

KDlamini committed:
Add styling for the header/navbar

KDlamini committed:
Update README screenshot with new png files in images folder

KDlamini committed:
Create a section classed 'main' that contains a header:
- Add a banner that will host an h1 and p element(s), and social media link.
- Apply all necessary images from the images folder.
- Add styles for all the header child element.

KDlamini committed:
fix stylelint errors in .css file

KDlamini committed:
Add newline after <html> tag in index.html file

KDlamini committed:
Add linters.yml Config file for GitHub Actions

KDlamini committed:
Rename LinkedIn and logo icon files to fix Webhint Report erros

KDlamini committed:
Update logo img src link and LinkedIn social media href link

KDlamini committed:
Add header background

AbdumurodovaZulfizar commented on README.md:
```diff
@@ -4,14 +4,20 @@
+
 
 ## Authors
 
```
**REQUIRED:** Please kindly personalize the MIT.md file, add your name, project name and delete unnecessary lines, so it will look more professional.

![image](https://user-images.githubusercontent.com/77872355/129119024-1fca7405-c38e-4483-81ad-532face61607.png)

AbdumurodovaZulfizar commented on index.html:
```diff
@@ -8,6 +8,37 @@
 </head>
 <body>
-    
+    <nav class="toolbar">
+        <figure class="logo-wrapper">
+            <img src="./images/My-Logo.png" alt="">
+        </figure>
+
+        <figure class="menu-icon">
+            <img src="./images/Union.png" alt="">
+        </figure>
+    </nav>
```
**REQUIRED:** The container of the toolbar section must be `header`, not `nav`, because as you can see toolbar is not parent of any link.

AbdumurodovaZulfizar commented on index.html:
```diff
@@ -8,6 +8,37 @@
 <body>
-    
+    <nav class="toolbar">
+        <figure class="logo-wrapper">
+            <img src="./images/My-Logo.png" alt="">
+        </figure>
```
**REQUIRED:** The logo should be `a` element, please try to put the logo image inside an `a` element, so whenever user clicks on logo in the future he/she will go back to the top of portfolio.

AbdumurodovaZulfizar commented on index.html:
```diff
@@ -8,6 +8,37 @@
+            <img src="./images/My-Logo.png" alt="">
+        </figure>
+
+        <figure class="menu-icon">
+            <img src="./images/Union.png" alt="">
+        </figure>
```
**REQUIRED:** The hamburger button should have `nav` tag, because you will use it for links to different parts of the page in the future.

AbdumurodovaZulfizar commented on index.html:
```diff
@@ -8,6 +8,37 @@
+                    don’t hestiate to contact me.
+                </p>
+                <a href="#">LET’S CONNECT</a>
+                <div class="social-media-icons">
+                    <a href="http://twitter.com"><img src="./images/twitter.svg" alt="" class="social-icon"></a>
+                    <a href="http://Linkedin.com"><img src="./images/Linkedin-icon.svg" alt="" class="social-icon"></a>
+                    <a href="http://medium.com"><img src="./images/medium.svg" alt="" class="social-icon"></a>
+                    <a href="http://github.com"><img src="./images/github.svg" alt="" class="social-icon"></a>
+                    <a href="http://angellist.com"><img src="./images/angellist.svg" alt="" class="social-icon"></a>
+                </div>
```
**REQUIRED:** You should use `ul` to group your icons, and `li` per icon. The main reason of is that you should practice semantic HTML as much as possible.

AbdumurodovaZulfizar commented on style.css:
```diff
@@ -0,0 +1,128 @@
+  margin: 0;
+  padding-top: 72px;
+}
+
+.toolbar {
+  position: fixed;
+  margin: 0;
+  height: 72px;
+  width: 95vw;
+  padding: 0 24px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+}
+
+.logo-wrapper {
+  height: 20px;
+  width: 76px;
+}
```
**REQUIRED:** My Logo has much more space on the left-hand side, make it the same with the template, please try to follow the **Figma** as much as possible.

See the difference:
![image](https://user-images.githubusercontent.com/77872355/129120273-75fe96ce-2a2b-46ec-b272-c7736a78e69e.png)
![image](https://user-images.githubusercontent.com/77872355/129120327-10b9a066-5fb3-4074-b897-8a3f0ff456cb.png)

AbdumurodovaZulfizar commented on style.css:
```diff
@@ -0,0 +1,128 @@
+  align-items: center;
+  justify-content: center;
+}
+
+.banner {
+  width: 63.888%;
+}
+
+.banner h1 {
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 60px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: left;
+}
+
```
**REQUIRED:** It is impossible to see your name on phone mode, the toolbar is on it. Please kindly fix this issue. Use inspector to see how your project look like on different sizes.
![image](https://user-images.githubusercontent.com/77872355/129120166-f267d18d-cd39-4019-ad6a-eca9d68948a2.png)

AbdumurodovaZulfizar commented on style.css:
```diff
@@ -0,0 +1,128 @@
+  align-items: center;
+  justify-content: center;
+}
+
+.banner {
+  width: 63.888%;
+}
+
```
**REQUIRED:** The banner has much more space on the left and right because of setting its width to 63.888%, instead of it use margin and padding.

![image](https://user-images.githubusercontent.com/77872355/129120760-5e4755a3-e48b-46fe-940d-010285e04638.png)
![image](https://user-images.githubusercontent.com/77872355/129120835-a9f0e05f-e9ed-4184-9b64-7dc0465abc11.png)


**REQUIRED:** There is a little difference between the font colour of this section, it would be a good idea if you change it.

AbdumurodovaZulfizar reviewed (changes requested):
Hello @KDlamini, I am @AbdumurodovaZulfizar, your reviewer for this project :wave:

Congratulations on doing an awesome project 💥
**TO HIGHLIGHT:**

- You have set up linters correctly and there are no errors
- You have followed GitHub flow and the feature branch has a descriptive name
- You have written descriptive comments with the correct tense
- You have used `flex` to align elements

But, to master your work, there is some work to do, I strongly offer you to implement the given design as much as possible, also use semantic HTML a bit more.

### **CHANGES REQUIRED ♻️ 🛑**


**Look through the comments below 👇 for required changes**


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
<hr>

Happy Coding 🔥🔥🔥
The review was done by @AbdumurodovaZulfizar
If there is anything that is not 💯% clear don't hesitate to hit me on slack @Zulfizar Abdumurodova or leave messages on PR.

<hr>

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

KDlamini committed:
Personalize the MIT.md file, add your name, project name and delete unnecessary lines

KDlamini committed:
Update .html and .css file for the header element:
- Change logo to a clickable link using an anchor element.
- Use nav element for menu items. menu items must be a list.
- Update styles for logo and menu items.

KDlamini committed:
Style menu icon and display on mobile view:
- Make menu a button
- Update media queries for smartphones

KDlamini committed:
Fix all lint errors in .css file

KDlamini committed:
Re-arrange css lines to make code look pretty

KDlamini committed:
Modify header background.
- Update styles for header text

KDlamini committed:
Update mobile background media queries.
- Change h1 element text
- Add img element for mobile background image
- Fix responsive design irregularities

KDlamini committed:
Modify headers h1 element text

KDlamini committed:
Fix margin issues for banner text

KDlamini committed:
Update css with flexbox to improve responsive design.
- Remove unwanted images and their relative img tags.
- Update media queries relative to new css for heading and toolbox.

DanteAlonsoHT commented on README.md:
````diff
@@ -4,14 +4,33 @@
 
 - Html/Css
 
+## Additional Tools
+
+- Figma
+- stylelint
+
+## Getting Started
+
+To get a local copy of this project:
+
+Clone this repository or download the Zip folder:
+```
+git clone git@github.com:KDlamini/portfolio-website.git
+```
+## Live Demo
+
+[Preview](https://kdlamini.github.io/portfolio-website/)
 
 ## Authors
````
Your invitation to go to the `issue page` is missing, please don't forget that, to make a more professional README file.
Example: 
```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!
Feel free to check the [issues page](https://github.com/Your_Github_Account/Repository_Name/issues).
```
- [ ] Add the link to go to your `issue page` in your repository.

DanteAlonsoHT commented on style.css:
```diff
@@ -0,0 +1,275 @@
+  color: #505f79;
+}
+
+@media only screen and (max-width: 1200px) {
+  /* Tablets [601px -> 1200px] */
+}
```
- [ ] Remove empty @media if you are not using it.

DanteAlonsoHT commented on style.css:
```diff
@@ -0,0 +1,275 @@
+  }
+}
+
+@media only screen and (max-width: 425px) {
+  /* Small smartphones [325px -> 425px] */
+}
```
- [ ] Remove empty @media if you are not using it.

DanteAlonsoHT reviewed (changes requested):
<h1>Status: Changes Required 🚫 ♻️  </h1>

Hi @KDlamini, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ]  Your live demo link is not working, please fix it or remove it from your README file.

_Check the comments under the review._

## Optional suggestions

_Check the comments under the review to see if there are some optional suggestions._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

<hr>

You're almost done! I know you can do these changes required in time! 😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

KDlamini committed:
update README file.
- Remove leive demo link that is not working.
- Add a link to the issues page

KDlamini committed:
Remove empty @media in css file

mke2111 reviewed (approved):
### Hi @KDlamini, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tahiry-dev reviewed (approved):
Hello Simo Nkosi  👋, 

Congratulations! 🎉 , your project is complete!  You have done an amazing job 💪💪.
You can now merge it and move to the next step!, 
 👉keep up the good work

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
